plugins/scaffolder/src/next/EmbeddedScaffolderWorkflow/EmbeddedScaffolderWorkflow.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState, useCallback } from 'react';
+import type { ReactNode } from 'react';
+import { Box, Button } from '@material-ui/core';
+import { EmbeddableWorkflow, type WorkflowProps } from '../Workflow/Workflow';
+import { useGetCustomFields } from '../Router/Router';
+
+/**
+ * @alpha
```

Comment:
What is the reason to use `@alpha` here?

Reply:
@tudi2d this is my lack of knowledge about these directifves, what should it be?